expose.sh

```diff
@@ -1,14 +1,22 @@
 #!/usr/bin/env bash
 
+topdir=$(pwd)
+scriptdir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
```

Comment:
These two variable declarations were moved to the top since they're a "pair" and the `topdir` was needed when looking for a project config file.